### DISCUSSION TITLE
When shall we expect Paseto V4 Local?

### DISCUSSION THREAD

**Discussion by MarianoFacundoArch** [panva/paseto#54] [Q&A]
Is it something to be added?

**Comment by panva** [comment]
It is not. The cipher it uses has now dropped off the CFRG focus and it's not available in `node:crypto`.

**Comment by MarianoFacundoArch** [comment]
Why not implementing like paseto-ts did? They have implemented V4 for nodejs

**Reply by panva** [reply]
Because I believe in cryptography primitives implementation coming from the platform it runs on, not from an external dependency.
